_data.tf

```diff
@@ -12,18 +12,13 @@ variable "get_ip_command" {
   type = "map"
 
   default = {
-    ""    = ""
+    ""    = "/usr/bin/ifconfig enp2s0 | grep 'inet ' | cut -d: -f2 | awk '{print $2}'"
```

Comment:
I'd prefer to have something more descriptive than "". How about "bare", "metal" or "baremetal"?

Reply:
I think that as long as we run coreos and plug the network in the first slot, the name won't change:
> Names incorporating physical/geographical location of the connector of the hardware (example: enp2s0)
https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/

but I am happy to consider alternatives for getting that ip address. 

Now this `aws` and `gcp` are providers as opposed to the proposed names. So I tried to default running anywhere outside those 2 cloud providers.